src/common/com/intellij/plugins/haxe/model/HaxeMemberModel.java

```diff
@@ -97,6 +97,7 @@ public static HaxeMemberModel fromPsi(PsiElement element) {
     if (element instanceof HaxeVarDeclaration) return new HaxeFieldModel((HaxeVarDeclaration)element);
     if (element instanceof HaxeLocalVarDeclaration) return new HaxeLocalVarModel((HaxeLocalVarDeclaration)element);
     if (element instanceof HaxeParameter) return new HaxeParameterModel((HaxeParameter)element);
+    if (element instanceof HaxeForStatement) return null;
```

Comment:
Or... above, given that the review comments are in order of entry, not in order they appear in the sources.  :/